src/compat/assumptions.h

```diff
@@ -50,6 +50,7 @@ static_assert(sizeof(double) == 8, "64-bit double assumed");
 //             code.
 static_assert(sizeof(short) == 2, "16-bit short assumed");
 static_assert(sizeof(int) == 4, "32-bit int assumed");
+static_assert(sizeof(unsigned) == 4, "32-bit unsigned assumed");
```

Comment:
Wouldn't `unsigned int` be the value to test for equivalence?

Reply:
`unsigned` is a shorthand for `unsigned int`.

Reply:
I think this check is redundant anyway, because prepending a type with unsigned shouldn't change it's bitlength. But I didn't feel like looking that up and adding the check has no downsides.